plugins/parsers/json_v2/parser.go

```diff
@@ -586,25 +586,25 @@ func (p *Parser) convertType(input gjson.Result, desiredType string, name string
 		case "uint":
 			r, err := strconv.ParseUint(inputType, 10, 64)
 			if err != nil {
-				return nil, fmt.Errorf("Unable to convert field '%s' to type uint: %v", name, err)
+				return nil, fmt.Errorf("unable to convert field '%s' to type uint: %w", name, err)
```

Comment:
```suggestion
				return nil, fmt.Errorf("unable to convert field %q to type uint: %w", name, err)
```

Honestly, I think just `%s` would be sufficient as well for these..